src/vm/gcinfodecoder.cpp

```diff
@@ -1169,7 +1169,7 @@ void GcSlotDecoder::DecodeSlotTable(BitStreamReader& reader)
         m_SlotArray[0].Slot.RegisterNumber = regNum;
         m_SlotArray[0].Flags = flags;
 
-        UINT32 loopEnd = min(m_NumRegisters, MAX_PREDECODED_SLOTS);
+        UINT32 loopEnd = _min(m_NumRegisters, MAX_PREDECODED_SLOTS);
```

Comment:
Why do we need to use _min instead of min? If the min is not defined for the DAC build, I would define it conditionally just for DACCESS_COMPILE.